### PR TITLE
Fix #3009 Chosing non-existing css file breaks MOLGENIS

### LINKED ISSUE #3009: Chosing non-existing css file breaks MOLGENIS
Morris:"I managed to break molgenis01 because I chose a non-existent css file."

### PR DESCRIPTION
**Pull request by Mark-de-Haan**
* Fix website crashing when submitting a non existing css
* Fix theme manager showing duplicates in the dropdownlist

### CODE REVIEW

File: molgenis-core-ui/src/main/resources/css/themes/bootstrap-molgenis.css [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-	padding: 5px;
-}
-
-label.horizontal-inline {
-	margin-right: 5px;
-}
-
-label.horizontal-inline.inbetween {
-	margin-left: 5px;
-}
```
Comment (fdlk) [inline molgenis-core-ui/src/main/resources/css/themes/bootstrap-molgenis.css]: I think I'd like to keep the source around in case we want to change it? :)

Comment (Mark-de-Haan) [inline molgenis-core-ui/src/main/resources/css/themes/bootstrap-molgenis.css]: http://unminify.com/?
I was thinking the same thing but then i thought how little effort it is to unminify it.
Reason for removing is the ResourcePath object we use to generate a list of styles uses the molgenis.css as well as the generated molgenis.min.css

Creating two molgenis options in the user interface

File: pom.xml [hunk 2]
```diff
@@ -529,11 +529,14 @@
                                 <webappTargetDir>${project.build.directory}/generated-resources</webappTargetDir>
                                 <cssSourceIncludes>
                                     <cssSourceInclude>**/*.css</cssSourceInclude>
+                                    <cssSourceInclude>**/**/*.css</cssSourceInclude>
                                 </cssSourceIncludes>
                                 <cssSourceExcludes>
                                     <!-- changes to excludes must be applied to molgenis-core-ui/src/main/resources/templates/resource-macros.ftl -->
                                     <cssSourceExclude>**/*.min.css</cssSourceExclude>
                                     <cssSourceExclude>**/*-min.css</cssSourceExclude>
+                                    <cssSourceExclude>**/**/*.min.css</cssSourceExclude>
```
Comment (fdlk) [inline pom.xml]: Above comment line suggests that resource-macros.ftl also needs to be changed then?

File: molgenis-core-ui/src/main/resources/css/themes/bootstrap-molgenis.css (hunk 1 above)
Comment (fdlk) [inline molgenis-core-ui/src/main/resources/css/themes/bootstrap-molgenis.css]: I think I got the deduplication running in a different way, plus added unit tests for the style service. What do you think of https://github.com/Mark-de-Haan/molgenis/pull/10 ?

**Pull request closed without merging**